app/controllers/v1/registrations_controller.rb

```diff
@@ -1,6 +1,10 @@
 class V1::RegistrationsController < ApplicationController
+  skip_before_filter :authenticate, only: [:create]
```

Comment:
[SUG] as we're using `before_action`... we could move to `skip_before_action` too...